lib/Doctrine/ORM/AbstractQuery.php

```diff
@@ -112,7 +112,7 @@ abstract class AbstractQuery
     /**
      * The map of query hints.
      *
-     * @var array
+     * @psalm-var array<string, mixed>
```

Comment:
What's the reason to use psalm-var instead of var for this kind of notation here and on several other places?

Reply:
To precise my question. I know not every IDE understands this syntax but the most used one PHPStorm does meanswhile.
Also I saw that in some changes of this PR the syntax is also used without the specific psalm-prefix.

Reply:
I'm myself not super interested in this since I don't use PHPStorm, so I try not to break auto-completion for people that do by using the prefix. I think it might be easier to search and replace `psalm-var` with `var` later on than the other way around.

>  Also I saw that in some changes of this PR the syntax is also used without the specific psalm-prefix.

Feel free to make a PR getting this exactly right once I have finished the batches.